projects/v3/src/app/pages/auth/auth-direct-login/auth-direct-login.component.ts

```diff
@@ -225,8 +225,13 @@ export class AuthDirectLoginComponent implements OnInit {
       return this.navigate(['auth', 'registration', res.data.user.email, res.data.user.key]);
     }
 
+    let errorMessage = $localize`Your link is invalid or expired.`;
+    if (res.message == 'User not enrolled in any experience') {
+      errorMessage = $localize`User not enrolled in any practera experience`;
```

Comment:
hi @sasangachathumal @shawnm0705 can we use the error message directly from the error response?

Reply:
Sure. I have updated the code. to use api error response message.